dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx

```diff
@@ -143,71 +150,40 @@ const NewRunsComponent = () => {
                 <Th width={35}>{columnNames.result}</Th>
                 <Th width={25}>{columnNames.endtime}</Th>
                 <Th width={20}></Th>
+                <Th width={5}></Th>
                 <Th width={2}></Th>
               </Tr>
             </Thead>
-
-            {initNewRuns.map((item, rowIndex) => {
-              const rowActions = moreActionItems(item);
-              const isItemFavorited = !!item?.metadata?.[USER_FAVORITE];
-              const isItemSeen = !!item?.metadata?.[DASHBOARD_SEEN];
-              return (
-                <Tbody key={rowIndex} isExpanded={isRunExpanded(item)}>
+            <Tbody>
+              {initNewRuns.map((item, rowIndex) => {
+                const rowActions = moreActionItems(item);
```

Comment:
Is there a reason not to in-line this value and omit the local variable?

If not, then also consider converting this code block to an expression (i.e., after in-lining the value and removing the local variable, remove the `return` statement, too).

Likewise for the `SavedRunsComponent`, too.

Reply:
Local variable is needed to define the Actions column

Reply:
What I was thinking of is replacing line 178 with this:
```
                      rowActions={moreActionItems(item)}
```
which means that you don't need the `rowActions` variable assigned at line 160.

Local variables are useful for several things (this is not an exhaustive list):
- giving a "name" to a complicated expression, to make it easier to understand
- cacheing the result of an expression, to avoid recalculating it or to ensure that the same value is used more where it is referenced
- to improve the layout of the code on the page (e.g., to shorten long lines, to avoid awkward line wrapping, or to simplify subsequent expressions)

However, we need to recognize that there are costs to creating local variables:
- the assignments take up space on the page
- they potentially obfuscate the underlying value
- there might be runtime costs

So, if we're going to create a local variable, then the benefit needs to outweigh the cost; however, in this case, I don't think it does:
- the value is used only once (so there is no value to caching or avoiding repetition)
- the expression being "named" is not complicated, and the new name is not significantly different from the old one (in any way which conveys more meaning or provides better clarity)
- replacing the expression with the local variable at the point where it is referenced doesn't change the layout on the page
- so there is no point to wasting the extra line
- using the variable creates an extra, unneeded level of indirection which a reader or debugger will have to traverse if there is a question about the value
- (I can't easily tell if there are runtime costs in this instance)

So, I would omit it (which then enables another simplification).  But, perhaps you can just keep this in mind for future changes.